.vscode/launch.json

```
@@ -0,0 +1,31 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Launch Package",
+            "type": "go",
+            "request": "launch",
+            "mode": "auto",
+            "program": "${workspaceFolder}/main.go",
+            "args": ["--kubeconfig=${workspaceFolder}/kubeconfig_cluster1", "controller", "--namespace=open-cluster-management"],
```

kubeconfig_cluster1_e2e
```suggestion
            "args": ["--kubeconfig=${workspaceFolder}/kubeconfig_cluster1_e2e", "controller", "--namespace=open-cluster-management"],
```
kubeconfig_cluster1 didn't work well for me I couldn't get "ns" maybe role issue

---

I just tried it before submitting this PR and it ran. What error are you getting?

---

Seems to be working for me also. (In general we shouldn't be using the `*_e2e` kubeconfigs to run the controller since if there is an issue with RBAC, we should address that instead.)